js_modules/dagster-ui/packages/ui-core/src/search/useGlobalSearch.tsx

```diff
@@ -174,6 +259,11 @@ type IndexBuffer = {
   cancel: () => void;
 };
 
+// This is the version of the secondary query, used as part of the cache key.
+// When the data in the cache must be invalidated, this version should be bumped to prevent
+// fetching stale data.
+export const SEARCH_SECONDARY_DATA_VERSION = 'v1;';
```

Comment:
We recently added caching to store the latest asset query result to load the search UI when the query hasn't completed. This PR adds additional fields to that query, but this causes an issue on the first load as the logic assumes those fields exist but they don't on the cached query data. This causes the asset search UI to be unloadable.

This PR adds a version to the key to ensure that we won't fetch stale data.